File: week_02/day_01/multiple_classes/src/modules/car.py
class Car:
    def __init__(self, colour, model, engine, gearbox):
        self.colour = colour
        self.model = model
        self.engine = engine
        self.gearbox = gearbox
        self.miles = 0


    def add_miles(self, go):
        if go == True:
            self.miles += 5

File: week_13/day_02/CodeClan_Cars/src/main/java/vehicle/Car.java
package vehicle;

import vehicle.components.Engine;
import vehicle.components.GearBox;
import vehicle.components.Tyres;

public class Car extends Vehicle {
    private double milesPerGallon;

    public Car(Engine engine, GearBox gearBox, Tyres tyres, String colour, double price, double milesPerGallon) {
        super(engine, gearBox, tyres, colour, price);
        this.milesPerGallon = milesPerGallon;
    }

    @Override
    public double defaultPrice() {
        return 0;
    }

    @Override
    public double priceFor(Vehicle vehicle) {
        return 0;
    }
}
